### PR TITLE
fea(storybook): documentation additions

### PR DESCRIPTION
**Pull request by agliga**
## Description
* Finished adding all events
* Added tagsToString fixes for non reprated attribute tags
* Added tagsToString for some most components
* Added `text` doc
* Added `spread` attribute to button (just for a sample of how it should be, if it's good then will add to all components.)

## References
https://github.com/eBay/ebayui-core/issues/1494

### CODE REVIEW

**Review by austingardner — APPROVED** [review]
LGTM!

File: .storybook/storybook-code-source/index.js [hunk 1]
```diff
@@ -1,28 +1,39 @@
-function tagToString(tagName, args, fromArray = false) {
-    const { body, attrs } = argsToString(args);
+function tagToString(tagName, args, plurarls = {}, indent = '', fromAttribute = false) {
```
Comment (austingardner) [inline .storybook/storybook-code-source/index.js:1]: is `plurarls` for args to subcomponents? Maybe that name could be a little clearer

Comment (agliga) [reply .storybook/storybook-code-source/index.js:1]: Yes, it's basically to turn plural argument tags to singular
So `@items` becomes `@item`

**Review by ianmcburnie — APPROVED** [review]
Nice! Storybook is MUCH improved from our v7 branch :-)

**Pull request merged**